iidm/iidm-api/src/main/java/com/powsybl/iidm/network/OperationalLimits.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ * @author Thibaut Vermeulen <thibaut.vermeulen at rte-france.com>
+ */
+public interface OperationalLimits {
+    /**
+     * Get the operational limits' type (can be APPARENT_POWER, CURRENT or VOLTAGE)
+     */
+    LimitType getLimitType();
+
+    default void remove() {
+        // do nothing
+    }
```

Comment:
I'm not convinced you have to provide a default method here: default method should not be used to factorize the code (use abstract class instead).

Reply:
It is mostly to prevent breaking changes. Is it better in your opinion to throw an exception?